Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverOnboarding.yaml

```diff
@@ -807,4 +808,67 @@ HyperVergeSdkLogs:
         and: [driverId, docType]
       orderBy:
         field: createdAt
-        order: asc
\ No newline at end of file
+        order: asc
+
+
+HyperVergeVerification:
+  tableName: hyperverge_verification
+  fields:
+    id: Id HyperVergeVerification
+    documentImageId1: Id Image
+    documentImageId2: Maybe (Id Image)
+    driverId: Id Person
+    requestId: Text
+    docType: DocumentType
+    vehicleCategory: Maybe VehicleCategory
```

Comment:
Shouldn't maybe.

Reply:
as discussed as req type is maybe and it will be  better if we keep these maybe as well.